### DISCUSSION TITLE
Nominate a leading device for Notifications

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#953] [Ideas]
It's frustrating to receive a notification on every device for Element when I'm already at my computer looking at Element... If we nominate a 'lead' device that mute other deivces when you're on it that might be a way to get around this (similar to slack) 

Interesting screenshots/ideas below:

<img width="463" alt="Screenshot 2023-02-02 at 09 20 45" src="https://user-images.githubusercontent.com/89144281/216283425-b1d1a7f1-4235-498a-aceb-9da846220783.png">

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-meta/issues/360
